heareval/predictions/task_predictions.py

```diff
@@ -925,6 +1028,8 @@ def task_predictions(
         seed_everything(42, workers=False)
 
     metadata = json.load(embedding_path.joinpath("task_metadata.json").open())
+    metadata["mode"] = "folds"  # remove me, only for testing
```

Comment:
What is the convention that was defined? Now I'm forgetting

Reply:
I don't think we have a convention defined yet - need to check this b/c metadata["mode"] might not actually be defined for the existing open tasks

Reply:
Updated this so if "fold" is in metadata (and is a list of fold string names) then it will perform k-fold. Otherwise will work as normal with pre-defined splits.